opentelemetry-api/src/opentelemetry/context/threadlocal_context.py

```diff
@@ -23,14 +23,20 @@ class ThreadLocalRuntimeContext(RuntimeContext):
     implementation is available for usage with Python 3.4.
     """
 
+    class _Token:
```

Comment:
Is this class really necessary? I think we can just use `Context` instead of wrapping a `Context` object inside a "token" one.

Reply:
I agree philosophically, although there's probably something in the spec that states that the token should not be the context. Although I don't see anything stating that now: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/context.md#attach-context

Reply:
If so, then at least the underscore should be in `self._context` and not in `_Token`. We are returning an object, that makes its class part of a public API.

Reply:
Updated the class name